alerts/cluster-network-operator/OVNKubernetesControllerDisconnectedSouthboundDatabase.md

```diff
@@ -0,0 +1,53 @@
+# OVNKubernetesControllerDisconnectedSouthboundDatabase
+
+## Meaning
+
+The `OVNKubernetesControllerDisconnectedSouthboundDatabase` alert is triggered
+when the OVN controller is not connected to OVN southbound database
+for more than 5 minutes.
+
+## Impact
+
+Networking control plane is degraded on the node. Existing workloads
+on the node may continue to have connectivity but any networking configuration
+update will not be applied.
+
+## Diagnosis
+
+Find the ovnkube-node pod running on the affected node.
+
+```shell
+pod=$(oc get pod -n openshift-ovn-kubernetes -l app=ovnkube-node \
+  -o jsonpath={..metadata.name} --field-selector spec.nodeName=<node>)
+```
+
+Check the southbound database connection status.
+
+```shell
+oc exec -n openshift-ovn-kubernetes ${pod} -c ovn-controller -- ovn-appctl connection-status
+```
+
+Check the logs of the ovnkube-node container looking for ovn controller error logs.
```

Comment:
Since it is the `ovnkube-node` container that does the actual checking and alerting these are the logs I had in mind:
https://github.com/ovn-org/ovn-kubernetes/blob/64a5aea9d1e44cdf3f4651fe7fbc61968d999570/go-controller/pkg/metrics/ovn.go#L387-L396